.docker/vimrc

```diff
@@ -0,0 +1,69 @@
+colorscheme elflord
```

Comment:
I have no idea what this whole config does to `vim` - personally I use `vi` due to a former live in  SCO OpenServer. Will running `vi` on ubuntu boot that or `vim` with this config in place?

Reply:
Also, Docker containers are supposed to be single purpose, right - why do we need `vim` anyway?